server/src/main/java/org/opensearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequest.java

```diff
@@ -537,6 +555,16 @@ public RestoreSnapshotRequest source(Map<String, Object> source) {
                 } else {
                     throw new IllegalArgumentException("malformed ignore_index_settings section, should be an array of strings");
                 }
+            } else if (name.equals("storage_type")) {
```

Comment:
Out of the functional scope of this PR and more on syntax - can we convert these 10+(and more in future) if-else statements to switch case instead? Would lead to faster execution time by preparing the jump table and also make it more readable.

Reply:
Do we plan to fix this in this PR?